doc/modules/preprocessing.rst

```diff
@@ -388,9 +388,10 @@ values. However, this comes at the price of losing data which may be valuable
 i.e., to infer them from the known part of the data.
 
 The :class:`Imputer` class provides basic strategies for imputing missing
-values, either using the mean, the median or the most frequent value of
-the row or column in which the missing values are located. This class
-also allows for different missing values encodings.
+values, either using the mean, the median, the most frequent value of
```

Comment:
"values, either" -> "values. It can use ... Alternatively it can fill with the mean of only the k-nearest ..."